src/components/Notifications/NotificationsFeed.tsx

```diff
@@ -0,0 +1,253 @@
+            <div>
+              <Button inverted size="small" onClick={onBegin}>
+                {i18N[locale].onboarding.button}
+              </Button>
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className="dcl notifications-feed">
+      <div className="dcl notifications-feed__header">
+        <p className="dcl notifications-feed__title">
+          {i18N[locale].feed.title}
+        </p>
+      </div>
+      {!isLoading && (
+        <div className="dcl notifications-feed__content">
+          {userNotifications.length > 0 && (
+            <Tabs>
+              <Tabs.Tab
+                active={activeTab === 'newest'}
+                onClick={(e) => onChangeTab(e, 'newest')}
+              >
+                {i18N[locale].feed.tabs.newest}
+              </Tabs.Tab>
+              <Tabs.Tab
+                active={activeTab === 'read'}
+                onClick={(e) => onChangeTab(e, 'read')}
+              >
+                {i18N[locale].feed.tabs.read}
+              </Tabs.Tab>
+            </Tabs>
+          )}
+          {userNotifications.length > 0 && (
+            <div className="dcl notifications-feed__list-container">
+              <div className="dcl notifications-feed__list">
+                {activeTab == 'newest' ? (
+                  <>
+                    <div>
+                      {unreadNotifications.map((notification) => (
+                        <NotificationHandler
+                          notification={notification}
+                          locale={locale}
+                        />
+                      ))}
+                    </div>
+                    {previousNotifications.length > 0 && (
+                      <div>
+                        <p
+                          style={{
+                            fontSize: '16px',
+                            fontWeight: 600,
+                            paddingLeft: '16px',
+                            marginBottom: 0
+                          }}
+                        >
+                          Previous
+                        </p>
+                        {previousNotifications.map((notification) => (
+                          <NotificationHandler
+                            notification={notification}
+                            locale={locale}
+                          />
+                        ))}
+                      </div>
+                    )}
+                  </>
+                ) : (
+                  <>
+                    {readNotifications.map((notification) => (
+                      <NotificationHandler
+                        notification={notification}
+                        locale={locale}
+                      />
+                    ))}
+                  </>
+                )}
+              </div>
+            </div>
+          )}
+          {!userNotifications.length && (
+            <div className="dcl notifications-feed__emptyview">
```

Comment:
if you use the before suggestion change from `emptyview` to `empty-view` then this should be
```suggestion
            <div className="dcl notifications-feed__empty-view">
```

Reply:
keep it all as `notifications-feed__emptyview-*`